bigquery_etl/alchemer/survey.py

```diff
@@ -35,17 +36,27 @@ def format_responses(s, date):
     # Note that we are omitted date_submission and date_completed because the
     # timezone is not ISO compliant. The submission date that is passed in as
     # the time parameter should suffice.
-    fields = [
-        "id",
-        "session_id",
-        "status",
-        "response_time",
-    ]
+    fields = ["id", "session_id", "status", "response_time"]
+    results = []
+    for data in s.get("survey_data", {}).values():
+        # There can be answer_id's like "123456-other"
+        if data.get("answer_id") and isinstance(data["answer_id"], str):
+            numeric = re.findall(r"\d+", data["answer_id"])
+            if numeric:
+                data["answer_id"] = int(numeric[0])
+            else:
+                del data["answer_id"]
+        if data.get("options"):
+            data["options"] = list(data["options"].values())
+        if data.get("subquestions"):
+            data["subquestions"] = json.dumps(data["subquestions"])
```

Comment:
This is handled like a string because there are two types of subquestions that have different level of nesting.